### ISSUE TITLE
Better Anchor Tags

### ISSUE THREAD

**Issue by thisguyisbarry** [pvme/pvme.github.io#12]
I noticed some pages don't have anchor tags for a lot of the content that would be desirable to have quick access to. 
I encountered this issue on the abyssal creatures page, but it's pretty common throughout the website. 
https://pvme.github.io/pvme-guides/slayer/abyssal-creatures/

I think an ideal solution to this would be to have smaller, slightly indented text for headings currently using a <strong> tag (bolded and underlined text when viewed from a browser) since they're essentially subheadings. 
There'll also be some work in proper markdown for what would be considered a heading, but that's an issue for the actual guide repo. 

Happy to try and help on this if I can. There's currently a warning about the guide structure not sure how much that would effect working on this repo. (Maybe it does since there hasn't been a commit in almost 2 weeks?) 

Thanks!, 
Barry.

**Comment by thisguyisbarry** [comment]
Do you have an example of misuse? 
Feel like if I find anything that breaks to issue a pull request on the markdown to fix the misuse. What do you think?

Have those changes happened? (I don't really play RS anymore so don't keep us as much with specific communities)

**Comment by Towsti** [comment]
Quite a few guides use bold + underline for inline formatting: https://pvme.github.io/pvme-guides/basic-guides/araxxor-basic/
![image](https://user-images.githubusercontent.com/73656340/200560375-bf1c9b44-2425-477f-82f6-c52e52bf03bc.png)

Best way to go about that is probably to just create sub-headers when the bold/underline text is the only text on a line. 

The structure changes are all done by now

**Comment by thisguyisbarry** [comment]
Yeah, good point. I guess I won't try to inforce my will onto the world this time :) 

I'll look into this later and maybe message you. Would discord be a better place to message you? thisguyisbarry#1988
